calico/getting-started/kubernetes/windows-calico/maintain.mdx

```diff
@@ -1,8 +1,14 @@
 ---
-description: Tasks to manage Calico services and uninstall Calico for Windows.
+description: Tasks to manage Calico services and uninstall Calico for Windows when the manual install (non-HPC) method was used.
 ---
 
-# Start and stop Calico for Windows services
+# Manage and maintain manually installed {{prodnameWindows}} services
```

Comment:
Heading # cannot use variable. Change to: Manage manually-installed Calico for Windows services

Reply:
I had checked locally and it worked fine, just double checked on the netlify preview and the page renders correctly: https://deploy-preview-984--tigera.netlify.app/calico/latest/getting-started/kubernetes/windows-calico/maintain

Any specific reason why it can't be used? (as a side note, it also works on links)

Reply:
Sorry I didn't explain...heading #s are used in the TOC. 
<img width="318" alt="image" src="https://github.com/tigera/docs/assets/48458529/c94a505f-d6f8-4e5b-9639-7c8ea32ca8e6">

Reply:
Oh, I see, I had missed that, sorry. Will change those back!